samples/integration/cmake-vcpkg/.gitignore

```diff
@@ -0,0 +1,2 @@
+build 
+CMakeUserPresets.json
```

Comment:
why this file?

Reply:
The end of the guide talks about how to properly work on a project what uses this system with other collaborators and references using a git repo to do so. In this scenario you don't want your build directory or your `CMakeUserPresets.json` file pushed to the repo. You don't want the build dir pushed for reasons like someone else is working on a different platform, and you don't want `CMakeUserPresets.json` push because it contains file paths specific to your env.